test/components/DayPickerRangeController_spec.jsx

```diff
@@ -2698,18 +2698,6 @@ describe('DayPickerRangeController', () => {
       expect(modifiers.has('selected-span')).to.equal(true);
     });
 
-    it('contains `last-in-range` if this.isLastInRange returns true', () => {
```

Comment:
The removal of this test indicates it’s a breaking change; can we avoid that?

Reply:
I can revert the code deleted on [`this.modifier`](https://github.com/airbnb/react-dates/pull/1538/files/34ded9e0d6685c34b324ba7b51d562816100c3ee#diff-a5cfcc83315d3bea09a9b06437d7ee80L201) but keep the deleted [last in range check and styles add](https://github.com/airbnb/react-dates/pull/1538/files/34ded9e0d6685c34b324ba7b51d562816100c3ee#diff-e5503a8207c75f758aa5c4cbe229a678L330) that caused the issue